src/genetics.rs

```diff
@@ -13,12 +13,23 @@ use rand::{distributions::Uniform, thread_rng, Rng};
 use rayon::iter::Zip;
 use rayon::prelude::*;
 use rayon::vec::IntoIter;
+use static_assertions::const_assert_eq;
 
-pub const POPULATION: usize = 100;
-const NUM_SURVIVORS: usize = POPULATION / 2;
-const NUM_PARENT_PAIRS: usize = NUM_SURVIVORS / 2;
-const NUM_CHILDREN_PER_PARENT_PAIRS: usize = POPULATION / NUM_PARENT_PAIRS;
-const MUTATION_RATE: u8 = 5; // Percent
+pub const POPULATION: usize = 1000;
+const SELECTION_RATE: f32 = 0.1;
+const MUTATION_RATE: f32 = 0.01;
+
+#[allow(
+    clippy::cast_sign_loss,
+    clippy::cast_possible_truncation,
+    clippy::cast_precision_loss
+)]
+const SURVIVORS: usize = ((POPULATION as f32) * SELECTION_RATE) as usize;
+const_assert_eq!(100, SURVIVORS);
```

Comment:
I don't understand why you want these asserts?

Reply:
I'll change it up tomorrow, they served two purposes - first as an assurance that all the type conversions remained valid despite the clippy warnings, and the second was so I had an idea of the effect changing the "independent" consts had on these "dependent" consts.

I'm glad to see the changes here significantly improved the time to solve the default problem, but I wasn't satisfied with the values as-is because it would seem `Al Escargot` was never getting solved. I also didn't know yet that restarting is a valid strategy so I played with the explosion rate being less than a 100% wipe, just a major shakeup but it didn't seem to help.

I expected these to go away anyway when these values become user input instead of consts 👍